.infrastructure/confgiMap.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+ name: app-config
+ namespace: todoapp
+data:
+ PYTHONUNBUFFERED: 3.2
```

Comment:
The value for `PYTHONUNBUFFERED` should be `1` to ensure that Python's output is unbuffered. This is important for real-time logging and debugging. Please change `3.2` to `1`.